plugin/pkg/auth/authenticator/request/x509/x509.go

```diff
@@ -92,7 +92,10 @@ var CommonNameUserConversion = UserConversionFunc(func(chain []*x509.Certificate
 	if len(chain[0].Subject.CommonName) == 0 {
 		return nil, false, nil
 	}
-	return &user.DefaultInfo{Name: chain[0].Subject.CommonName}, true, nil
+	return &user.DefaultInfo{
```

Comment:
@liggitt we have some additional checks in ours and we separate this case from `SubjectToUserConversion`, but I can't say that I remember why.